src/Tests/dotnet-run.Tests/GivenDotnetRunRunsCsProj.cs

```diff
@@ -566,7 +566,7 @@ public void ItForwardsEmptyArgumentsToTheApp()
 
             new DotnetCommand(Log, "run")
                 .WithWorkingDirectory(testInstance.Path)
-                .Execute("a", "", "c")
```

Comment:
is -- required now? It used to be optional

Reply:
Just switched it back to being optional: https://github.com/dotnet/sdk/pull/14379/commits/5927d4ccad0aa47cfccb95ba5c8a6b6b0e6decbe#diff-c59ee7ae48403cce44671ddc3549eb921e7c23bda26840c883b29d9f7e7942e5R36